monitor/views.py

```diff
@@ -106,7 +108,7 @@ def monitor_home(request):
 
 @cache_page(60 * 60 * 24)
 def sounds_stats_ajax(request):
-    last_week = datetime.datetime.now()-datetime.timedelta(weeks=1)
+    last_week = datetime.datetime.now()-datetime.timedelta(weeks=2)
```

Comment:
I think all these variables called `last_week` which now cover 2 weeks should be renamed. Maybe something more generic like `time_period` or `time_span` or something like this.